src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    private String email;
```

Comment:
never forget about regular `id` field
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;    
    
    @Column(unique = true)
    private String email;
```